src/app/api/tasks/tasks.service.ts

```diff
@@ -472,7 +483,7 @@ export class TasksService extends BaseService {
   private sendCompanyTaskNotifications = async (
     task: Task,
     notificationService: NotificationService,
-    isReassigned = false,
+    _isReassigned = false, // someday this will come in handy
```

Comment:
where is this used? I don't see it in the code..

Reply:
It isn't now, but it might be used in the future, just putting this out here to remind me where to look for :stuck_out_tongue: